testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -37,6 +37,7 @@
     SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR(fallback -> fallback.test(SUPPORTS_TOPN_PUSHDOWN) && fallback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY)),
 
     SUPPORTS_AGGREGATION_PUSHDOWN,
+    SUPPORTS_AGGREGATION_PUSHDOWN_SUM_DISTINCT(SUPPORTS_AGGREGATION_PUSHDOWN),
```

Comment:
What is this for?

(i don't see usages that would require a new TCB)

Reply:
Replaced it with `SUPPORTS_AGGREGATION_PUSHDOWN`